### PR TITLE
All heard messages are now autocapitalized

### LINKED ISSUE #3023: Non-human mobs don't have autocapitalization
As a human, saying things automatically capitalizes the first word of your sentence.
As a simple mob, cyborg, or brainmob, you lack this capitalization.

Intended, ya or na?

### PR DESCRIPTION
**Pull request by Crazylemon64**
Fixes #3023 
This is supposed to be something that's wanted to be fixed, right

### CODE REVIEW

File: code/modules/mob/hear_say.dm [hunk 1]
```diff
@@ -9,6 +9,8 @@
 			//Or someone snoring.  So we make it where they won't hear it.
 		return
 
+	message = capitalize(message)
```
Comment (tigercat2000) [inline code/modules/mob/hear_say.dm]: nononononononononononononononononononononononononononononononononononono 
jesus christ no

if you absolutely fucking have to, put this at https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/say.dm#L121

Comment (Crazylemon64) [inline code/modules/mob/hear_say.dm]: Alright, this is clearly not the way to go about this. I'll shut this ill-conceived PR down, then.

Comment (Crazylemon64) [inline code/modules/mob/hear_say.dm]: Out of curiosity, how much, and in what manner, would this have reduced everything everyone knew and loved to a pile of scattered ashes?

Comment (tigercat2000) [inline code/modules/mob/hear_say.dm]: @Crazylemon64 It probably would have _functioned_
but dear god making all incoming messages automatically capitialized screams "WHY"

**Pull request closed without merging**